packages/flutter_tools/lib/src/macos/darwin_dependency_management.dart

```diff
@@ -87,6 +96,22 @@ class DarwinDependencyManagement {
       // whether to run.
       useCocoapods = _plugins.isNotEmpty;
     }
+
+    final Event event = Event.flutterInjectDarwinPlugins(
+      platform: platform.name,
+      isModule: _project.isModule,
```

Comment:
This will always be false because we return on line 81 if it's a module

Reply:
Right, this is meant to future proof the logic for when we do support modules. I think that's preferable to explicitly set this to `false`, but let me know if feel otherwise.

Reply:
Nit: I would move this above `final bool useCocoapods;`. It's kind of breaking up the logic of setting up the Podfile (based on the `useCocoapods` var.

Reply:
I moved this to the bottom so it's tucked away from the rest of the logic!